client/components/CategoryCompanyGender.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import {PieChart, Pie, Tooltip, Cell, Legend, Label} from 'recharts'
+import {connect} from 'react-redux'
+import {getCompanyGenderStats} from '../actions/stats/getCategoryCompanyGender'
+
+const colors = ['#3fb1c8', '#c84e4e']
+
+export class CategoryCompanyGender extends React.Component {
+  componentDidMount () {
+    this.props.dispatch(getCompanyGenderStats(3))
```

Comment:
use `Number(props.match.params.id)` instead of `3`

Reply:
done